databricks_template_schema.json

```diff
@@ -316,6 +331,44 @@
           }
         ]
       }
+    },      
+    "input_docker_image": {
```

Comment:
Does this need to be an initialization parameter (generally we set this when it's used across several files) or can this be a different kind of parameter (job parameter, manually copy-pasted, etc.)?

Reply:
good catch, I will probably remove it as a parameter and document that it needs to be replaced later by the user

Reply:
The parameter is actually used in all the gitlab pipelines files.
The docker image is the image for a VM to run the CI/CD with all the required libraries . That is a best practice in gitlab as opposite to adding the commands to install the dependencies in the pipelines themselves, for the following reasons:
- you do not need to re-install the dependencies for each pipeline execution
- installing the dependencies on-the-fly in gitlab pipeline seems more complicated as you cannot leverage on tools like github actions/checkout

Reply:
I've updated the file
` template/{{.input_root_dir}}/docs/mlops-setup.md.tmpl`
 to describe the process to create and push the docker image

Reply:
Got it, but in most cases, we're still recommending to the user to provide a default TODO value at time of initialization since they won't have the image ready? At that point, they'd have to manually update all the spots with the image any way right? So wouldn't it be simpler to mark those spots with a TODO instead?

Reply:
I agree if we assume they wouldn't have an image ready. 
But they might have the image ready, especially after they run their first project, they are likely to be able to reuse the same image.

Reply:
Publicly available image as default would be great! And in that case, we can leave as a parameter since it will only show up as an extra knob for GitLab but can still work right out of the box

Reply:
Hmm this seems like a temporary solution, I'm ok with it if we can come up with a long-term solution for this to be a Databricks hosted image

Reply:
The plan is to get a Field engineering account on Docker Hub for Databricks. Or use the Databricks runtime one. 

Then we'll just need to update the default docker image in this project.

Reply:
Sounds good, can you file a follow-up on your end to track this? Thanks!